build-system/release-tagger/create-release.js

```diff
@@ -37,19 +38,64 @@ const {GraphQlQueryResponseData} = require('@octokit/graphql'); //eslint-disable
  * @return {string}
  */
 function _formatPullRequestLine(pr) {
-  const {mergeCommit, number, title, url} = pr;
+  const {mergeCommit, title} = pr;
   const {abbreviatedOid, commitUrl} = mergeCommit;
   return dedent`\
-  <a href="${commitUrl}"><code>${abbreviatedOid}</code></a> ${title} \
-  (<a href="${url}">#${number}</a>)`;
+   <a href="${commitUrl}"><code>${abbreviatedOid}</code></a> - ${title}`;
+}
+
+/**
+ * Organize bento changes into sections
+ * @typedef {{
+ *  [major: string]: {
+ *    packages: {
+ *      [extension: string]: Set
+ *    }
+ *    unchanged: Set,
+ *  }}} Metadata
```

Comment:
Worth moving the typedef to the top of this file in case `Metadata` might need be used elsewhere in future? And if you do move it, maybe call the type `PackageMetadata` to help disambiguate?